modules/services/mpd.nix

```diff
@@ -162,25 +160,24 @@ in {
         Environment = "PATH=${config.home.profileDirectory}/bin";
         ExecStart = "${cfg.package}/bin/mpd --no-daemon ${mpdConf}";
         Type = "notify";
-        ExecStartPre = ''${pkgs.bash}/bin/bash -c "${pkgs.coreutils}/bin/mkdir -p '${cfg.dataDir}' '${cfg.playlistDirectory}'"'';
+        ExecStartPre = ''
+          ${pkgs.bash}/bin/bash -c "${pkgs.coreutils}/bin/mkdir -p '${cfg.dataDir}' '${cfg.playlistDirectory}'"'';
```

Comment:
and such change in nixpkgs can cause a mass rebuild and break things dramatically.

Reply:
Hmm, I'm not certain why this would cause a mass rebuild? As far as I can tell they are semantically identical. The only change seems to be the initial newline and indentation, which should be stripped by Nix.

Reply:
That said, I think this code could be made more clear by using `pkgs.writeScript`.